### PR TITLE
Redux with React components ⚛️🔬

### PR DESCRIPTION
**Pull request by rmrmrmr**
## On this pull request we:

- Added three books to the initial state 📖
- Applied inmutability to redux reducers 🧪

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    {
+      id: 'dVxxLqI4ueuoOcRkmd1cP',
+      title: 'The Banquet',
+      author: 'Plato',
+      category: 'Philosophy',
+      chapter: 'Chapter 1',
+      percentage: 12,
+    },
+  ],
 };
```
Comment (zieeco) [inline src/redux/books/books.js:30]: ### FORM

- Please, your form gets submitted when the input is empty this makes users add empty content and it gets added to the book. Kindly ensure the form is not submitted on the empty input field to improve your app's user experience.

Comment (zieeco) [reply src/redux/books/books.js:30]: ### README 

- Please, let's use the currently updated readme [template](https://github.com/microverseinc/readme-template) to update our readme so it is consistent with the new one also kindly remove all the placeholder text as they're there to guide you on how to update and add your personal content.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @rmrmrmr ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @rmrmrmr 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

giphy (1)

## Highlights

✅ Good Pull request description
✅ Good work making the changes requested by previous reviewer 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**